src/helpers/metrics.js

```diff
@@ -0,0 +1,37 @@
+import throttle from "lodash/throttle";
+import { AMPLITUDE_KEY } from "constants/env";
+
+const METRICS_ENDPOINT = "https://api.amplitude.com/2/httpapi";
+let cache = [];
+
+const uploadMetrics = throttle(() => {
```

Comment:
[Amplitude's SDK is like 60kb](https://bundlephobia.com/result?p=amplitude-js@6.2.0)! We just need some dead simple metrics with no user tracking, so I threw together a shitty little uploader to hit the HTTP API directly.